laws/src/main/scala/org/http4s/laws/discipline/ArbitraryInstances.scala

```diff
@@ -264,7 +264,7 @@ private[http4s] trait ArbitraryInstances {
     }
 
   implicit val http4sTestingCogenForContentCoding: Cogen[ContentCoding] =
-    Cogen[String].contramap(_.coding)
+    Cogen[String].contramap(_.coding.map(_.toUpper.toLower))
```

Comment:
why toUpper.toLower?

Reply:
It makes it consistent with the equalsIgnoreCase. The validation we do restricts us to charsets where it doesn't matter, but the types say it can, and I'd rather not tell a second lie.